### ISSUE TITLE
Log output is confusing

### ISSUE THREAD

**Issue by Mygod** [shadowsocks/simple-obfs#8]
```
# ss-server
 2017-01-09 17:20:03 INFO: using tcp fast open
 2017-01-09 17:20:03 INFO: onetime authentication enabled
 2017-01-09 17:20:03 INFO: plugin "obfs-server -p 8139 --obfs http --fast-open" enabled
 2017-01-09 17:20:03 INFO: initializing ciphers... chacha20
 2017-01-09 17:20:03 INFO: tcp port reuse enabled
 2017-01-09 17:20:03 INFO: listening at 0.0.0.0:49078
 2017-01-09 17:20:03 INFO: running from root user
 2017-01-09 17:20:03 INFO: using tcp fast open
 2017-01-09 17:20:03 INFO: obfuscating enabled
 2017-01-09 17:20:03 INFO: tcp port reuse enabled
 2017-01-09 17:20:03 INFO: listening at 0.0.0.0:8139
 2017-01-09 17:20:03 INFO: running from root user
^C 2017-01-09 17:23:14 ERROR: plugin service exit unexpectedly
```

As you can see here that one can't distinguish between ss-server's log and obfs-server's log.

Btw why Ctrl+C will it show that error?

**Comment by madeye** [comment]
Let me refine the plugin's output.

The error means plugin stops before ss-server. Maybe I should only show this message if exitcode of plugin is non zero.

**Comment by madeye** [comment]
Fixed via f2357e8
